# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/glibc23/gcc33-hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

Rene Rebe: This fixes compilation with gcc-3.3

--- ./stdio-common/sscanf.c.orig	2003-12-18 01:03:48.000000000 -0300
+++ ./stdio-common/sscanf.c	2003-12-18 01:04:03.000000000 -0300
@@ -29,8 +29,8 @@
 int
 sscanf (const char *s, const char *format, ...)
 {
-  va_list arg;
   int done;
+  va_list arg;
 
   va_start (arg, format);
   done = __vsscanf (s, format, arg);
